packages/botonic-plugin-google-translate/src/google-translate-api-service.ts

```diff
@@ -7,34 +7,75 @@ export interface LanguageDetection {
   confidence: number
 }
 
+export interface Translation {
+  translatedText: string
+  detectedLanguageCode: string
```

Comment:
isn't this redundant if we're setting language in Input?

Reply:
I know we don't use it but I wanted the API Service to return the same information that receives keeping the same structure.